modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalDisplayContext.java

```diff
@@ -1568,6 +1584,17 @@ private JournalDisplayContext(
 			_themeDisplay.getLocale(), _themeDisplay.getTimeZone());
 	}
 
+	private String _formatTooltipDate(Date date) {
```

Comment:
here we could better use _getFormatTooltipDate as the name of the method, so that we make clear what we are returning